charts/oes/templates/opa-deploy.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.opa.enabled }}
+apiVersion: extensions/v1beta1
```

Comment:
Should we update the apiVersion to apps/v1?

Reply:
Yes, updated the same